src/c_code_generator.js

```diff
@@ -2243,17 +2270,17 @@ module.exports = async function generate(rom, functionName, fileName, bFastMode,
             code += "    binaryAction.b = 0;\n";
             code += "    binaryAction.c = op;\n";
             code += "    binaryAction.opcode = 1;\n";
+            code += "    binaryAction.type = 2;\n";
             code += "    required.Binary.push_back(binaryAction);\n";
         }
 
-
+        
         if (rom.program[zkPC].arith == 1)
```

Comment:
with current versión of zkasmcom the arith flag no longer exists, instead of it use arithEq0 + arithEq1 + arithEq2 or arithEq0 || arithEq1|| arithEq2

Reply:
Fixed